### PR TITLE
Add `postgres` support into `seqr` app chart.

### PR DESCRIPTION
**Pull request by bpblanken**

### CODE REVIEW

File: .github/workflows/unit_tests.yaml [hunk 1]
```diff
@@ -57,12 +40,13 @@ jobs:
         run: python3 -m unittest unit_test/**/*.py
       - name: Test Seqr Chart Installation
         run: |
-          # This hack lets us skip some expensive setup within the seqr startup
```
Comment (bpblanken) [inline .github/workflows/unit_tests.yaml:60]: Note an example init for this with the helm chart install is here: https://stackoverflow.com/questions/55499984/postgresql-in-helm-initdbscripts-parameter

File: .github/workflows/unit_tests.yaml [hunk 2]
```diff
@@ -57,12 +40,11 @@ jobs:
...
-          export PGPASSWORD='super-secure-password'; psql -h localhost -U postgres -c 'CREATE DATABASE seqrdb'; psql -h localhost -U postgres -c 'CREATE DATABASE reference_data_db';
```
Comment (bpblanken) [inline .github/workflows/unit_tests.yaml:61]: I _think_ the correct usage of `startupProbe` added several months ago is rendering this unnecessary.  The standard flow (running all migrations + a few manage.py commands) works in under 2min!

File: charts/seqr/values.yaml [hunk 3]
```diff
@@ -179,6 +174,56 @@ redis:
...
+  architecture: standalone
+  auth:
+    username: postgres
+    existingSecret: *postgresSecretName
+    secretKeys:
+      # Keyword used to find the admin password inside the existingSecret
+      adminPasswordKey: password
+  fullnameOverride: seqr-postgresql
+  image:
+    tag: '12.19.0-debian-12-r9'
```
Comment (bpblanken) [inline charts/seqr/values.yaml:188]: The current cloudsql postgres version.

File: charts/seqr/values.yaml [hunk 4]
```diff
@@ -179,6 +174,56 @@ redis:
...
+    # Some details here: https://github.com/bitnami/charts/issues/14683
+    startupProbe:
+      enabled: true
+
+    # This is a reduced initContainers intended to replace the default
+    # provided with volumePermissions.enabled set to True.  The default
+    # uses the /data subdirectory and ignores postgresqlDataDir
+    # leading to an incorrectly permissioned data directory.
+    # See here: https://github.com/bitnami/charts/blob/aa038e9953df2ba74d481cb7ba1cccb3dc32c5fc/bitnami/postgresql/templates/primary/statefulset.yaml#L117-L157
+    initContainers:
```
Comment (bpblanken) [inline charts/seqr/values.yaml:206]: There was additional futzing to make the volume work correctly in a sane way.  By default, the PGDATA would be mounted to `/seqr/data`, and I thought it was nicer to have the directory include `postgresql` in some way.  `/seqr/postgresql-data` fits the pattern with the rest of our directory names.

File: charts/seqr/values.yaml [hunk 5]
```diff
@@ -179,6 +174,56 @@ redis:
     enabled: false
   fullnameOverride: "seqr-redis"
 
+postgresql:
+  enabled: false
```
Comment (hanars) [inline charts/seqr/values.yaml:178]: wouldn't it be better to enable this by default and just disable it for us in tg-0helm?

Comment (bpblanken) [reply charts/seqr/values.yaml:178]: 👍 yes, that works!  I was following the pattern of `redis`, but we can maybe just enable both of them in the open source chart? 

One other option is leaving disabled by here but enabling within the yet to exit `seer-platform` chart, 🤷

Comment (hanars) [reply charts/seqr/values.yaml:178]: okay yeah I did not realize redis was off by default. We should def have postgres and redis behave identically in the open source chart, up to you which approach you prefer

Comment (bpblanken) [reply charts/seqr/values.yaml:178]: I think it makes more sense to enable them both here and disable `postgresql` in our chart.

**Review by hanars — APPROVED** [review]
Approved, although we should probably make the corresponding tgg helm PR with redis/postgres disabled soon

**Pull request merged**